client/layout/index.jsx

```diff
@@ -411,9 +411,7 @@ class Layout extends Component {
 				{ this.props.shouldQueryAllSites && <QuerySites allSites /> }
 				<QueryPreferences />
 				<QuerySiteFeatures siteIds={ [ this.props.siteId ] } />
-				{ this.props.isUnifiedSiteSidebarVisible && (
```

Comment:
We need to query the site admin color (from the Jetpack site) only if we're showing the nav-unified sidebar (of a classic view sites); is there any reason why we dropped this condition in this PR?

Reply:
I want to prefetch the admin color when the GSV opens but the value is true only when the unified site sidebar is shown instead of the global one.